charger/phoenix-ev-eth.go

```diff
@@ -13,15 +13,11 @@ import (
 const (
 	phxEVEthRegStatus     = 100 // Input
 	phxEVEthRegChargeTime = 102 // Input
-	phxEVEthRegMaxCurrent = 528 // Holding
+	phxEVEthRegMaxCurrent = 300 // Holding
```

Comment:
den hast Du mir doch extra reinkorrigiert?

Reply:
Ich weiss. Scheint trotzdem nach Aktenlage falsch zu sein.
Die Doku ist da absolut widersprüchlich und mehrdeutig. Auch noch zwischen Englisch und Deutsch.
Im Register 528 steht jedenfalls i.d.R. 0 drin und in 300 z. B. 16.
Ich will es nochmal am lebenden Objekt testen wenn ich nochmal Gelegenheit dazu bekomme.
Alles noch wip...